task/power/src/main.rs

```diff
@@ -429,6 +429,7 @@ fn main() -> ! {
         hl::sleep_for(1000);
 
         let state = get_state();
+        let now = sys_get_timer().now;
```

Comment:
I feel like we ought to record this closer to where the measurement happens (i.e. down by `read_temperature` et al)

Reply:
Done!